firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -47,92 +47,120 @@
 // show us if debugging
 #ifdef PSYNC_DEBUG_FLAG
     #warning "PSYNC_DEBUG_FLAG defined"
-    #define DEBUG_PRINT(x)  Serial.println(x)
+    #define DEBUG_PRINT( x )  Serial.println( x )
 #else
-    #define DEBUG_PRINT(x)
+    #define DEBUG_PRINT( x )
 #endif
 
 // chip select pin for CAN Shield
-#define CAN_CS 10
+#define CAN_CS ( 10 )
 
 // ms
-#define PS_CTRL_RX_WARN_TIMEOUT (150)
-
+#define PS_CTRL_RX_WARN_TIMEOUT ( 150 )
 
+//
+static uint32_t last_update_ms;
 
 
 // *****************************************************
-// static global data
+// static structures
 // *****************************************************
 
 
-//
-static uint32_t last_update_ms;
-
-
 // construct the CAN shield object
-MCP_CAN CAN(CAN_CS);                                    // Set CS pin for the CAN shield
-
+MCP_CAN CAN( CAN_CS );                                    // Set CS pin for the CAN shield
 
 //
 static can_frame_s rx_frame_ps_ctrl_brake_command;
 
-
 //
 static can_frame_s tx_frame_ps_ctrl_brake_report;
 
-
 //
-static PID pidParams;
+static PID pid_params;
 
 
 // *****************************************************
 // static declarations
-// *
+// *****************************************************
 
 
-// corrects for overflow condition
-static void get_update_time_delta_ms(
-		const uint32_t time_in,
-		const uint32_t last_update_time_ms,
-		uint32_t * const delta_out )
+// *****************************************************
+// Function:    timer_delta_ms
+// 
+// Purpose:     Calculate the milliseconds between the current time and the
+//              input and correct for the timer overflow condition
+// 
+// Returns:     uint32_t the time delta between the two inputs
+// 
+// Parameters:  [in] timestamp - the last time sample
+// 
+// *****************************************************
+static uint32_t timer_delta_ms( uint32_t last_time )
 {
-    // check for overflow
-    if( last_update_time_ms < time_in )
-    {
-		// time remainder, prior to the overflow
-		(*delta_out) = (UINT32_MAX - time_in);
+    uint32_t delta = 0;
+    uint32_t current_time = millis( );
 
-        // add time since zero
-        (*delta_out) += last_update_time_ms;
-    }
-    else
+    if ( current_time < last_time )
     {
-        // normal delta
-        (*delta_out) = ( last_update_time_ms - time_in );
+        // Timer overflow
+        delta = ( UINT32_MAX - last_time ) + current_time;
+    }   
+    else
+    {   
+        delta = current_time - last_time;
     }
+    return ( delta );
+}
+
+
+// *****************************************************
+// local declarations
+// *****************************************************
+void wait_enter( );
+void wait_update( );
+void wait_exit( );
+
+void brake_enter( );
+void brake_update( );
+void brake_exit( );
+
+
+// *****************************************************
+// local definitions
+// *****************************************************
+int calculate_SLA_duty_cycle( float pre )
```

Comment:
Use int16_t for "int"s and int32_t for "long"s.  We want to be specific about the width of the variables.